### ISSUE TITLE
Append date and time to admin notices

### ISSUE THREAD

**Issue by kieran-wpmedia** [wp-media/wp-rocket#1867]
Feature request on behalf of a customer:

Problem:
It is not clear when an admin notice was generated or when an action (Preload, CPCSS etc) was completed

Solution:
Add a date and time stamp to the end of the message stating when the action was completed or message generated.

Link to request: https://secure.helpscout.net/conversation/913483428/117448/

**Comment by arunbasillal** [comment]
Something that came up in a discussion with @webtrainingwheels on what notices should and shouldn't have timestamps. 

- If a notice is displayed instantly because of an action triggered by the user (Clearing Cache manually, or saving a setting), that notice DOES NOT need a timestamp. 
- If the notice is triggered or updated automatically (preload update or preload complete, Critical Path CSS generation), then that NEEDS a timestamp. 

In other words, as long as the time of generation of the notice isn't current time, that needs a timestamp.
